bin/fasterer

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-$LOAD_PATH << File.expand_path('../../lib', __FILE__)
+$LOAD_PATH << File.expand_path('../lib', __dir__)
```

Comment:
I think this is not related to the main topic of this change, right?

Reply:
You are right, it is not. It's a tiny change suggested by rubocop, I couldn't resist adding this bit, thinking of it as undeserving of its own PR.

Reply:
Sure, and it's no my call to decide this;-)

It just kind of looked like it could be something you needed to adapt to get it working locally so I wanted to hint about this.